.gitignore

```diff
@@ -1,24 +1 @@
-# Compiled Object files, Static and Dynamic libs (Shared Objects)
-*.o
-*.a
-*.so
-
-# Folders
-_obj
-_test
-
-# Architecture specific extensions/prefixes
-*.[568vq]
-[568vq].out
-
-*.cgo1.go
-*.cgo2.c
-_cgo_defun.c
-_cgo_gotypes.go
-_cgo_export.*
-
-_testmain.go
-
-*.exe
-*.test
-*.prof
+vendor
```

Comment:
1) We are not checking in vendor folder
2) during development we have to test this client that time we have to install dependent packages in this folder and test.
3) To avoid excluding this directory each time I commit I added this folder name here.

Reply:
Thanks for the explanation!

So, why are we not checking in the vendor directory?

Reply:
Oops, sorry, I am confusing this with docker-infoblox. I agree you probably don't want to checkin the vendor directory.